src/sentry/testutils/outbox.py

```diff
@@ -65,19 +65,21 @@ def assert_webhook_payloads_for_mailbox(
     request: WSGIRequest,
     mailbox_name: str,
     region_names: list[str],
+    destination_types: dict[DestinationType, int] | None = None,
 ) -> None:
     """
     A test method for asserting that a webhook payload is properly queued for
      the given request
 
     :param request:
     :param mailbox_name: The mailbox name that messages should be found in.
-    :param region_names: The regions each messages should be queued for
+    :param region_names: Optional list of regions each messages should be queued for
+    :param destination_type: The destination type of the messages
```

Comment:
I think it's a mapping of destination type to the message count for the desintation? Probably worth including in this docstring.